its/src/test/java/com/sonar/it/csharp/Tests.java

```diff
@@ -91,8 +91,7 @@ public static void runNuGet(Orchestrator orch, Path projectDir, String... argume
   }
 
   private static Path getNuGetPath(Orchestrator orch) {
-    String toolsFolder = Paths.get("tools").resolve("nuget.exe").toAbsolutePath().toString();
```

Comment:
Location no longer exists (not related to this change) so this line is pointless.